File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        existing_max_wealth = 0
        for i in range(len(accounts)):
            total_money_for_each_customer = 0
            for j in accounts[i]:
                total_money_for_each_customer += j
            if  total_money_for_each_customer > existing_max_wealth:
                existing_max_wealth = total_money_for_each_customer
        return existing_max_wealth

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.ts
function maximumWealth(accounts: number[][]): number {
    let totalCustomersWealth = 0;
    for(let i=0; i<accounts.length; i++) {
        let totalAmount = 0;
        for(let j=0; j < accounts[i].length; j++) {
            totalAmount += accounts[i][j]
        }
        if(totalCustomersWealth < totalAmount) {
            totalCustomersWealth = totalAmount
        }
    }
    return totalCustomersWealth;
};
